ibm_mq/datadog_checks/ibm_mq/data/conf.yaml.example

```diff
@@ -140,6 +140,14 @@ instances:
     #   SYSTEM.*: queue_type:system
     #   DEV.*: role:dev,queue_type:default
 
+    ## @param version_output_cmd - string - optional - default: dspmqver
+    ## Custom command to run dspmqver to collect IBM MQ version if applicable. The agent will try to run
+    ## the `dspmqver` command by default to collect the version, but if the command doesn't work or another 
+    ## command is needed (e.g. docker exec ibm_mq dspmqver) the version will not be collected. This will not 
+    ## affect the collection of any metrics. 
+    #
+    # version_output_cmd: dspmqver
```

Comment:
We discussed previously about `MQCACH_REMOTE_VERSION`, but actually, getting it from the Queue Manager itself seems more appropriate. See Version section in : https://www.ibm.com/support/knowledgecenter/SSFKSJ_7.5.0/com.ibm.mq.ref.adm.doc/q087830_.htm 

You can get the version info using: 

```
mq_mgr_resp = pcf.MQCMD_INQUIRE_Q_MGR()
print(mq_mgr_resp[0][pymqi.CMQC.MQCA_VERSION])
> b'09010100'
```

The version format:

The version of the IBM WebSphere MQ installation, the queue manager is associated with. The version has the format VVRRMMFF:
VV: Version
RR: Release
MM: Maintenance level
FF: Fix level

Reply:
Ahh that makes things so much easier. I guess the main question I have with that is that the `MQCA_VERSION` was added with [pymqi 1.11.1](https://github.com/dsuch/pymqi/commit/4420ff0318b78662109334cd915e992e439fbf7a
). But I think I can still call on the key 2120 to get that version with:

```
print(mq_mgr_resp[0][2120]) 
b'09010100'
```

Do you think this is a suitable approach or should I assume that pymqi version is 1.11.1?

Reply:
@steveny91 You can upgrade the pymqi version to `1.11.1`.

https://github.com/DataDog/integrations-core/blob/master/ibm_mq/requirements.in#L2-L1

You will also need to `ddev dep freeze` to sync the agent requirements file.

Reply:
@AlexandreYang Thanks! I tried implementing that. I swapped the version in the requirements.in and then ran `ddev dep freeze` but the test seems to be failing on python2 while working on python3. I reverted my code to what was working before on pymqi 1.10.1 and tested it again and the test were still failing. Once I reverted back to what I had, all the test were passing again.